.fastly/config.toml

```diff
@@ -13,11 +13,8 @@ ttl = "5m"
     toolchain_constraint = ">= 1.17 < 1.19" # upper limit should be temporary (just while we figure out any rough edges)
 
   [language.rust]
-  toolchain_version = "1.56.1"
   toolchain_constraint = ">= 1.56.1"
   wasm_wasi_target = "wasm32-wasi"
-  fastly_sys_constraint = ">= 0.3.3"
-  rustup_constraint = ">= 1.23.0"
```

Comment:
The validation steps have been simplified and we no longer need to check the internal fastly-sys crate version, nor is the deprecated `toolchain_version` needed to be kept. We also don't need to check for all three binaries (rustc, cargo _and_ rustup) as the official recommended installation is to use rustup and that comes with cargo (which we check for as the primary toolchain tool).